lib/virtfs/xfs.rb

```diff
@@ -1 +1,20 @@
 require "virtfs/xfs/version"
+require_relative "xfs/allocation_group"
+require_relative "xfs/bmap_btree_block"
+require_relative "xfs/bmap_btree_record"
+require_relative "xfs/bmap_btree_root_node"
+require_relative "xfs/directory"
+require_relative "xfs/directory2_data_header"
+require_relative "xfs/directory3_data_header"
+require_relative "xfs/directory_block_tail"
+require_relative "xfs/directory_data_header"
+require_relative "xfs/directory_entry"
+require_relative "xfs/inode"
+require_relative "xfs/inode_map"
+require_relative "xfs/short_form_directory_entry"
+require_relative "xfs/short_form_header"
+require_relative "xfs/superblock"
+
+require_relative "xfs/base"
+require_relative "xfs/file_class"
+require_relative "xfs/dir_class"
```

Comment:
Minor, but it's common in gems to just refer to file assuming lib is on the path.  That is, instead of `require_relative 'xfs/whatever'`, prefer `require 'virtfs/xfs/whatever'`